openapi/paths/storefront/cashier-deposit.yaml

```
@@ -0,0 +1,31 @@
+post:
+  x-products:
+    - Storefront
+  tags:
+    - Cashier
+  summary: Create a deposit
+  operationId: StorefrontPostCashierDeposit
+  x-sdk-operation-name: deposit
+  security:
+    - CustomerJWT: []
+  description: Create a cashier deposit transaction.
```

nitpick: based on our style
```suggestion
  description: Creates a cashier deposit transaction.
```